analyzers/src/SonarAnalyzer.Common/Rules/PropertyGetterWithThrowBase.cs

```diff
@@ -22,7 +22,7 @@ namespace SonarAnalyzer.Rules
 {
     public abstract class PropertyGetterWithThrowBase : SonarDiagnosticAnalyzer
     {
-        protected const string DiagnosticId = "S2372";
+        protected const string DiagnosticId = "S101";//"S2372";
```

Comment:
This doesn't seem intentional

Reply:
Well it was. ;) Otherwise I can not check if the VB tests run as well. But I'll roll it back. :)

Reply:
I've created RSPEC PR and pushed the scaffolding here

Reply:
Now I see. I changed this in the wrong file. Rolled it back.